src/components/car-class-details/single-class.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  getCarClassName,
+  parseIRating,
+} from "../utils/formatter/CarConversion";
+import { convertMsToDisplay } from "../utils/formatter/UnitConversion";
+import {
+  selectCurrentSession,
+  useSelector,
+  selectLiveTiming,
+  LiveTiming,
+} from "@/lib/redux";
+
+interface CarDetails {
+  name: string;
+  color: string;
+  count: number;
+  fastestLap: number;
+  fastestLapDriver: string;
+}
+
+interface SplitDetails {
+  iRatingTotal: number;
+  count: number;
+}
+
+export const SingleClassDetails = () => {
+  const session = useSelector(selectCurrentSession);
+  const standings = useSelector<LiveTiming[]>(selectLiveTiming);
```

Comment:
Yeah this is going to be lots of calculations every half second (or whatever update interval we decide on for standings). I have no prob merging as is and optimize later, though.